common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -863,6 +863,7 @@
     public static final KnownKey zimbra_deregistered_authtoken_queue_size = KnownKey.newKey(5000);
     public static final KnownKey zimbra_jwt_cookie_size_limit = KnownKey.newKey(4096);
     public static final KnownKey zimbra_authtoken_cookie_domain = KnownKey.newKey("");
+    public static final KnownKey zimbra_same_site_cookie = KnownKey.newKey("Strict");
```

Comment:
by default should the value be Strict or None? 
Since this fix is addressing this header not be used in zimbraReverseProxyResponseHeaders and there its not default, so should we keep it as Strict here

Reply:
It's better to go with Strict as default value, we don't want to compromise on security aspect
Just need to be sure that we are not regressing anything on the ui clients
we should document this if customer wants to go to earlier behaviour then they need to change value to None, although it's not recommended by Zimbra

Reply:
I agree the default value should be `Strict`.